app/src/org/commcare/activities/CommCareSetupActivity.java

```diff
@@ -384,6 +387,12 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
                     result = data.getStringExtra(InstallArchiveActivity.ARCHIVE_JR_REFERENCE);
                 }
                 break;
+            case GET_APPS_FROM_HQ:

```

Comment:
Should there be error messaging (Toast notification) in the event of the request failing?